### ISSUE TITLE
Star character in comment field 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [franca/franca#137]
```

If I use "*" in the comment, Franca editor reports an error.

method SendTone {
  in {
    <**
      @description : Accepted tones are: [0-9], [A-D], * oder #.
    **>
    UInt8 tone
  }
```

Original issue reported on code.google.com by `manfred....@bmw.de` on 2 Mar 2015 at 4:15

Attachments:
* [Star_character_in-comment-field-Franca_0.9.1.201412191134.png](https://storage.googleapis.com/google-code-attachments/franca/issue-137/comment-0/Star_character_in-comment-field-Franca_0.9.1.201412191134.png)

**Comment by kbirken** [comment]
Implementation hint: See how class FAnnotationAuxImpl is currently used to split the raw text of each FAnnotation element. This is added via genmodel.

**Comment by kbirken** [comment]
See also similar issue #173.

**Comment by kbirken** [comment]
Implemented by commit 0480c29 on develop.

The star character can be used now in the annotation comment by escaping it with a backslash: \*
When accessing the comment string via the generated EMF API, the escape backslash will be removed.
